joomla/joomla-cms
# [4.0] Implement the workflow into the blog sample data

Pull Request for Issue #21707 . (first part)

### Summary of Changes
This PR let the blog sample data creating a new Workflow to ensure, that the articles are published


### Testing Instructions
Apply the "Blog Sample data" and check the workflows, a new workflow called "Sample Workflow" is created and assigned to all new sample data categories.
